src/libraries/System.Net.Quic/src/PInvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,2 @@
+<!-- MsQuic currently not available -->
+msquic.dll!MsQuicOpen
```

Comment:
Can someone help me get this right? This inclusion isn't removing the warning for:
msquic.dll!MsQuicOpen is not supported on one\more targeted platforms.

Reply:
If the analyzer is not helping, you can disable it using `<EnablePInvokeAnalyzer>false</EnablePInvokeAnalyzer>`